sdk/logs/src/main/java/io/opentelemetry/sdk/logs/DefaultLogEmitter.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.logs;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.sdk.logs.data.Severity;
+import java.time.Instant;
+import java.util.concurrent.TimeUnit;
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * No-op implementations of {@link LogEmitter}.
+ *
+ * <p>This implementation should induce as close to zero overhead as possible.
+ */
+@ThreadSafe
+class DefaultLogEmitter implements LogEmitter {
+
+  private static final DefaultLogEmitter INSTANCE = new DefaultLogEmitter();
+
+  static LogEmitter getInstance() {
+    return INSTANCE;
+  }
+
+  @Override
+  public LogBuilder logBuilder() {
+    return new NoopSpanBuilder();
```

Comment:
Can use a singleton here instead of instantiating a new instance each time.

Reply:
done